### PR TITLE
Fix slider not zooming on drag

### PR DESCRIPTION
**Pull request by lchan131**

### CODE REVIEW

File: media-player.js [hunk 1]
```diff
@@ -701,6 +701,10 @@ class MediaPlayer extends FocusVisiblePolyfillMixin(InternalDynamicLocalizeMixin
 		return this._media ? this._media.textTracks : null;
 	}
 
+	get zoomBar() {
```
Comment (MichaelKovalchuk) [inline media-player.js:704]: i would make it a function that returns zoomBar

File: media-player.js [hunk 2]
```diff
@@ -2069,8 +2072,8 @@ class MediaPlayer extends FocusVisiblePolyfillMixin(InternalDynamicLocalizeMixin
 	}
 
 	_sliderChange() {
-		if (!this._zoomBar) return;
-		const zoomBarValue = this._zoomBar.immediateValue - 50;
+		if (!this.zoomBar) return;
```
Comment (MichaelKovalchuk) [inline media-player.js:2075]: if function is used - you can do something like this: 
const zoomBar = this.getZoomBar() or, alternatively:
```
const zoomBar = this.shadowRoot.getElementById('d2l-labs-media-player-zoom-bar');
if (!zoomBar) return;
const zoomBarValue = zoomBar.immediateValue - 50;
```

File: media-player.js (hunk 1 above)
Comment (lchan131) [reply media-player.js]: I just removed this getter

**Review by MichaelKovalchuk — APPROVED** [review]

File: media-player.js (hunk 2 above)
Comment (lchan131) [reply media-player.js]: I just took your advice and simply called getElementById.

**Pull request merged**